package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "can-event": "^3.6.0",
     "can-stache": "^3.7.3",
+    "can-test-helpers": "^1.0.1",
```

Comment:
this should be a devDependency.

Reply:
damn... thought I hit --save-dev. Good catch.